tests/acceptance/features/apiSpaces/uploadSpaces.feature

```diff
@@ -0,0 +1,50 @@
+@api @skipOnOcV10
```

Comment:
name `uploadSpaces.feature`. Maybe kind of like `upload data into spaces`?

Reply:
all our feature files have the scheme of camelCase and two words.
Should we change it now?